front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -110,14 +111,16 @@ const useScenarioData = (
   );
 
   // TODO Paced train : Adapt this to accept paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10613
-  useAutoUpdateProjection(infra, timetable.train_ids, trainScheduleSummaries);
+  useAutoUpdateProjection(
+    infra,
+    trainSchedulesResults.map((t) => t.id),
```

Comment:
Nit: this passes a new value to the hook each time a re-render happens. This happens to be mostly fine by chance because `useAutoUpdateProjection()` doesn't perform any heavyweight action when `editoastTrainIds` changes (apart from checking whether the selected/projected train still exist), but that may change in the future and will not be easily noticeable since it's a completely separate file.

I'd suggest a `useMemo()` to make sure the value is stable when the train schedules don't change.